lib/class-wp-rest-block-directory-controller.php

```diff
@@ -300,7 +300,11 @@ private static function parse_block_metadata( $plugin ) {
 			$block->assets[] = 'https://plugins.svn.wordpress.org/' . $plugin['slug'] . $asset;
 		}
 
-		$block->humanized_updated = human_time_diff( strtotime( $plugin['last_updated'] ), current_time( 'timestamp' ) ) . __( ' ago', 'gutenberg' );
+		$block->humanized_updated = sprintf(
+			/* translators: %s: Human-readable time difference. */
+			__( '%s ago', 'gutenberg' ),
+			human_time_diff( strtotime( $plugin['last_updated'] ), current_time( 'timestamp' ) )
```

Comment:
```suggestion
			human_time_diff( strtotime( $plugin['last_updated'] ) )
```

No need to provide the second argument since it's the default value. Feel free to ignore since slightly out of scope.